arkouda/pdarrayclass.py

```diff
@@ -3073,6 +3073,45 @@ def sqrt(pda: pdarray, where: Union[bool, pdarray] = True) -> pdarray:
     return power(pda, 0.5, where)
 
 
+@typechecked
+def skew(pda: pdarray) -> np.float64:
+
+    """
+    Computes the sample skewness of an array.
+    Skewness > 0 means there's greater weight in the right tail of the distribution.
+    Skewness < 0 means there's greater weight in the left tail of the distribution.
+    Skewness == 0 means the data is normally distributed.
+    Based on the `scipy.stats.skew` function.
+
+    Parameters
+    ----------
+    pda : pdarray
+        A pdarray of values that will be calculated to find the skew
+
+    Returns
+    -------
+        np.float64
+            The skew of all elements in the array
+
+    Examples:
+    >>> a = ak.array([1, 1, 1, 5, 10])
+    >>> ak.skew(a)
+    0.9442193396379163
+    """
+
+    deviations = pda - pda.mean()
+    cubed_deviations = deviations ** 3
+
+    std_dev = pda.std()
+
+    if std_dev != 0:
+        skewness = cubed_deviations.mean() / (std_dev ** 3)
+    else:
+        skewness = 0
+
+    return skewness
```

Comment:
completely unnecessary stylistic suggestion to cut down on number of lines, feel free to ignore :)
```python
cubed_deviations = deviations ** 3
std_dev = pda.std()
return 0 if std_dev == 0 else cubed_deviations.mean() / (std_dev ** 3)
```